extensions/sql-migration/src/api/azure.ts

```diff
@@ -83,6 +91,14 @@ export async function getBlobContainers(account: azdata.Account, subscription: S
 	return blobContainers!;
 }
 
+export async function getMigrationControllers(account: azdata.Account, subscription: Subscription, resourceGroupName: string, regionName: string): Promise<azureResource.MigrationController[]> {
+	const api = await getAzureCoreAPI();
+	let result = await api.getMigrationControllers(account, subscription, resourceGroupName, regionName, true);
+	let controllers = result.controllers;
+	sortResourceArrayByName(controllers!);
+	return controllers!;
```

Comment:
The typing currently says that this could be undefined so it'd be best if you handled it correctly in that case - up to you whether you do that in the azurecore side (always send an array - defaulting to empty if the response comes back with nothing)